### DISCUSSION TITLE
Chia 1.3.0 application will not start

### DISCUSSION THREAD

**Discussion by maepyon3** [Chia-Network/chia-blockchain#10889] [Support]
hello,

I upgraded from chia 1.2.0 to chia 1.3.0, but the application will not start.
I uninstalled everything and installed chia1.3.0, but it will not start.
I uninstalled everything and reinstalled chia 1.2.2, but after the startup loading screen appeared, the application stopped working.

Environment: Windows 10

How can I run it?

Thank you very much.

maepyon3

**Comment by loppefaaret** [comment]
One thing you can try, is to do a reset of the working directory of chia
If you manually move out the folder under your users home folder `C:\Users\<USERNAME>\.chia` and do a fresh install of version 1.3.1 - it should start working - it will require you to resync the chain from 0, and redo any settings, such as plot directories etc.
